docs/configuration.rst

```diff
@@ -39,6 +39,11 @@ A few relevant settings for your `Pyramid .ini file <http://docs.pylonsproject.o
         # Default: [r'^/static/?', r'^/api-docs/?']
         pyramid_swagger.exclude_paths = [r'^/static/?', r'^/api-docs/?']
 
+        # Definable contextmanager to handle request/response validation
```

Comment:
I don't think this will actually work here, since you need to pass a real python object in. Give the example in the python codeblock below?